### PR TITLE
Jetpack: Normalize package names

### PR DESCRIPTION
**Pull request by kraftbj**
Normalizes package names to automattic/jetpack

#### Changes proposed in this Pull Request:

* n/a

#### Jetpack product discussion

* n/a

#### Does this pull request change what data or activity we track or use?

* n/a

#### Testing instructions:

* n/a
*

#### Proposed changelog entry for your changes:

* n/a

### CODE REVIEW

**Comment by kraftbj** [comment]
This is fine on the Jetpack side. Marked as in progress since it'll need a little massage on the Fusion side.

File: projects/plugins/jetpack/_inc/lib/debugger/debug-functions.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /**
  * WP Site Health functionality temporarily stored in this file until all of Jetpack is PHP 5.3+
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/_inc/lib/debugger/debug-functions.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/class.jetpack-bbpress-json-api.compat.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /**
  * Deprecated. Moved to /3rd-party/ directory.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/class.jetpack-bbpress-json-api.compat.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/class.jetpack-twitter-cards.php [hunk 3]
```diff
@@ -2,7 +2,7 @@
 /**
  * Jetpack Twitter Card handling.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/class.jetpack-twitter-cards.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/class.photon.php [hunk 4]
```diff
@@ -2,7 +2,7 @@
 /**
  * Class for photon functionality.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/class.photon.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/json-api-config.php [hunk 5]
```diff
@@ -2,7 +2,7 @@
 /**
  * Config for the WP.com REST API
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/json-api-config.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/comments.php [hunk 6]
```diff
@@ -10,7 +10,7 @@
  * Feature: Engagement
  * Additional Search Queries: comments, comment, facebook, twitter, google+, social
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/comments.php:13]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/comments/comments.php [hunk 7]
```diff
@@ -6,7 +6,7 @@
 /**
  * Main Comments class
  *
- * @package JetpackComments
+ * @package automattic/jetpackComments
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/comments/comments.php:9]:
```suggestion
 * @package automattic/jetpack

File: projects/plugins/jetpack/modules/photon.php [hunk 8]
```diff
@@ -11,7 +11,7 @@
  * Feature: Recommended, Appearance
  * Additional Search Queries: photon, photo cdn, image cdn, speed, compression, resize, responsive images, responsive, content distribution network, optimize, page speed, image optimize, photon jetpack
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/photon.php:14]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/publicize.php [hunk 9]
```diff
@@ -11,7 +11,7 @@
  * Feature: Engagement
  * Additional Search Queries: facebook, jetpack publicize, twitter, tumblr, linkedin, social, tweet, connections, sharing, social media, automated, automated sharing, auto publish, auto tweet and like, auto tweet, facebook auto post, facebook posting
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/publicize.php:14]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/theme-tools/devicepx.php [hunk 10]
```diff
@@ -9,7 +9,7 @@
  *
  * add_theme_support( 'jetpack-devicepx' );
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/theme-tools/devicepx.php:12]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/theme-tools/site-breadcrumbs.php [hunk 11]
```diff
@@ -9,7 +9,7 @@
  * License: GPL2 or later
  * Text Domain: jetpack
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/theme-tools/site-breadcrumbs.php:12]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/widgets/milestone.php [hunk 12]
```diff
@@ -4,7 +4,7 @@
  *
  * Everything happens within the folder, but Jetpack loads widgets via a widgets/*.php scheme.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/widgets/milestone.php:7]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/widgets/milestone/class-milestone-widget.php [hunk 13]
```diff
@@ -2,7 +2,7 @@
 /**
  * Milestone Countdown Widget
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/widgets/milestone/class-milestone-widget.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/widgets/milestone/milestone.php [hunk 14]
```diff
@@ -2,7 +2,7 @@
 /**
  * Milestone Widget Loader.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/widgets/milestone/milestone.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-admin.php [hunk 15]
```diff
@@ -2,7 +2,7 @@
 /**
  * WordAds Admin.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-admin.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-api.php [hunk 16]
```diff
@@ -2,7 +2,7 @@
 /**
  * The WordAds API.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-api.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-california-privacy.php [hunk 17]
```diff
@@ -2,7 +2,7 @@
 /**
  * CCPA Class
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-california-privacy.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php [hunk 18]
```diff
@@ -2,7 +2,7 @@
 /**
  * CCPA Do Not Sell Widget
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-cron.php [hunk 19]
```diff
@@ -2,7 +2,7 @@
 /**
  * WordAds cron tasks.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-cron.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-params.php [hunk 20]
```diff
@@ -2,7 +2,7 @@
 /**
  * WordAds Param Class file.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-params.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/class-wordads-sidebar-widget.php [hunk 21]
```diff
@@ -2,7 +2,7 @@
 /**
  * Widget for adding ads to a sidebar.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/class-wordads-sidebar-widget.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/modules/wordads/php/networks/amazon.php [hunk 22]
```diff
@@ -2,7 +2,7 @@
 /**
  * Amazon Network.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/wordads/php/networks/amazon.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/e2e/plugins/e2e-plan-data-interceptor.php [hunk 23]
```diff
@@ -6,7 +6,7 @@
  * Version: 1.0.0
  * Text Domain: jetpack
  *
- * @package jetpack-test-plugin-e2e-plan-data
+ * @package automattic/jetpack-test-plugin-e2e-plan-data
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/e2e/plugins/e2e-plan-data-interceptor.php:9]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/e2e/plugins/e2e-plugin-updater.php [hunk 24]
```diff
@@ -6,7 +6,7 @@
  * Version: 1.0.0
  * Text Domain: jetpack
  *
- * @package jetpack-test-plugin-e2e-plugin-updater
+ * @package automattic/jetpack-test-plugin-e2e-plugin-updater
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/e2e/plugins/e2e-plugin-updater.php:9]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/php/3rd-party/test_class-jetpack-modules-overrides.php [hunk 25]
```diff
@@ -2,7 +2,7 @@
 /**
  * Tests the Module Override functionality.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/php/3rd-party/test_class-jetpack-modules-overrides.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/php/general/test-class.jetpack-network.php [hunk 26]
```diff
@@ -2,7 +2,7 @@
 /**
  * Tests the Jetpack_Network class.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/php/general/test-class.jetpack-network.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/php/general/test_class.jetpack-user-agent.php [hunk 27]
```diff
@@ -2,7 +2,7 @@
 /**
  * Tests for Jetpack's legacy User Agent class.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/php/general/test_class.jetpack-user-agent.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/php/test_class.jetpack_photon.php [hunk 28]
```diff
@@ -2,7 +2,7 @@
 /**
  * Tests for Jetpack's Photon class.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/php/test_class.jetpack_photon.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: projects/plugins/jetpack/tests/php/uninstall/test_uninstall.php [hunk 29]
```diff
@@ -2,7 +2,7 @@
 /**
  * Tests the uninstall functionality.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline projects/plugins/jetpack/tests/php/uninstall/test_uninstall.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: tools/docker/config/wp-tests-config.php [hunk 30]
```diff
@@ -2,7 +2,7 @@
 /**
  * A wp-config for testing.
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline tools/docker/config/wp-tests-config.php:5]:
```suggestion
 * @package automattic/jetpack
```

File: tools/docker/mu-plugins/avoid-plugin-deletion.php [hunk 31]
```diff
@@ -7,7 +7,7 @@
  * Author URI: https://automattic.com/
  * Text Domain: jetpack
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline tools/docker/mu-plugins/avoid-plugin-deletion.php:10]:
```suggestion
 * @package automattic/jetpack
```

File: tools/docker/mu-plugins/debug.php [hunk 32]
```diff
@@ -7,7 +7,7 @@
  * Author URI: https://automattic.com/
  * Text Domain: jetpack
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline tools/docker/mu-plugins/debug.php:10]:
```suggestion
 * @package automattic/jetpack
```

File: tools/docker/mu-plugins/fix-monorepo-plugins-url.php [hunk 33]
```diff
@@ -7,7 +7,7 @@
  * Author URI: https://automattic.com/
  * Text Domain: jetpack
  *
- * @package Jetpack.
+ * @package automattic/jetpack.
```
Comment (jeherve) [inline tools/docker/mu-plugins/fix-monorepo-plugins-url.php:10]:
```suggestion
 * @package automattic/jetpack
```

**Review by jeherve — CHANGES_REQUESTED** [review]

**Review by ebinnion — APPROVED** [review]
Assuming that the changes in `projects/plugins/jetpack/tests/e2e/plugins/e2e-plan-data-interceptor.php` and `projects/plugins/jetpack/tests/e2e/plugins/e2e-plugin-updater.php` are expected, this LGTM.

**Review by jeherve — APPROVED** [review]

**Comment by kraftbj** [comment]
r220186-wpcom

**Pull request merged**